src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
I would suggest to go with the zend-diactoros way than passing a string. It helps to pass files also  https://github.com/zendframework/zend-diactoros/blob/b4e7758556c97b5bb9a5260d898e9788ee800538/src/Stream.php#L36 . What is the real benefit of not going the way ?